packages/frontapp/src/app/components/BookInfo/Description.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC, useCallback, useState } from 'react';
+import { OpenLink, Topic } from './index';
+import styled from '@emotion/styled';
+import { colors, dimensions } from '@mimir/ui-kit';
+
+export const Description = styled.p`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+
+  font-weight: 300;
+  font-size: ${dimensions.base};
+  line-height: ${dimensions.xl};
+  color: ${colors.main_black};
+`;
+
+interface IProps {
+  children: React.ReactNode;
+}
+
+const DescriptionDetails: FC<IProps> = ({ children }) => {
+  const [showDescription, toggleDescription] = useState<boolean>(false);
+
+  const toggle = useCallback(() => {
+    toggleDescription(!showDescription);
+  }, [showDescription]);
```

Comment:
`toggle` method is not needed.
Rename `toggleDescription` to `setShowDescription` and call it  from onClick